### PR TITLE
New Adapter: Trustedstack

### PR DESCRIPTION
**Pull request by product-trustedstack**
Trustedstack server adapter added.

### CODE REVIEW

File: src/test/java/org/prebid/server/bidder/trustedstack/TrustedstackBidderTest.java [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    }
+
+    @Test
+    public void makeBidsShouldReturnCorrespondingMtypesAndAdTypes() throws JsonProcessingException {
+        final List<Bid> bids = new ArrayList<>();
+        final Bid bid1 = Bid.builder().impid("imp_id").mtype(1).build();
+        final Bid bid2 = Bid.builder().impid("imp_id").mtype(2).build();
+        final Bid bid3 = Bid.builder().impid("imp_id").mtype(3).build();
+        final Bid bid4 = Bid.builder().impid("imp_id").mtype(4).build();
+        bids.add(bid1);
```
Comment (SerhiiNahornyi) [inline src/test/java/org/prebid/server/bidder/trustedstack/TrustedstackBidderTest.java:125]: We would prefer to have separate test for each mtype

**Review by SerhiiNahornyi — CHANGES_REQUESTED** [review]

File: src/test/java/org/prebid/server/bidder/trustedstack/TrustedstackBidderTest.java (hunk 1 above)
Comment (product-trustedstack) [reply src/test/java/org/prebid/server/bidder/trustedstack/TrustedstackBidderTest.java]: Done.

Comment (product-trustedstack) [reply src/test/java/org/prebid/server/bidder/trustedstack/TrustedstackBidderTest.java]: @SerhiiNahornyi requesting review as the requested changes has been implemented.

File: src/main/java/org/prebid/server/spring/config/bidder/TrustedstackConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                .withConfig(trustedstackConfigurationProperties)
+                .usersyncerCreator(UsersyncerCreator.create(externalUrl))
+                .bidderCreator(config ->
+                        new TrustedstackBidder(resolveEndpoint(config.getEndpoint(), externalUrl), mapper))
+                .assemble();
+    }
+
+    private String resolveEndpoint(String configEndpoint, String externalUrl) {
+        return configEndpoint.replace(EXTERNAL_URL_MACRO, HttpUtil.encodeUrl(externalUrl));
+    }
```
Comment (AntoxaAntoxic) [inline src/main/java/org/prebid/server/spring/config/bidder/TrustedstackConfiguration.java:47]: URL resolving is usually a part of Bidder code, so please move it there

Comment (b-chinmay) [reply src/main/java/org/prebid/server/spring/config/bidder/TrustedstackConfiguration.java]: Done.

Comment (product-trustedstack) [reply src/main/java/org/prebid/server/spring/config/bidder/TrustedstackConfiguration.java]: Done.

**Comment by product-trustedstack** [comment]
@SerhiiNahornyi  requesting to share your approval if all looks good

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**